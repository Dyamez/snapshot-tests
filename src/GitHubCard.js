import './App.css';

export default function GitHubCard() {
  return (
    <div className="GitHubCard">
      <h1>GitHubCard Snapshot Challenge</h1>
      <p>Github User Name: Dyamez</p>
      <img src="./zenitsu.jpg" alt="pic">Image</img>
    </div>
  );
}
