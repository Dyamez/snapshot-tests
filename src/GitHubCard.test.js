import renderer from 'react-test-renderer'
import { render, screen } from '@testing-library/react';
import GitHubCard from './GitHubCard';

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard />).toJSON()
  expect(tree).toMatchSnapshot()
})