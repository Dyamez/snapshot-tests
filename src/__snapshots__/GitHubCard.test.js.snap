// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="GitHubCard"
>
  <h1>
    GitHubCard Snapshot Challenge
  </h1>
  <p>
    Github User Name: Dyamez
  </p>
  <img
    alt="pic"
    src="./zenitsu.jpg"
  >
    Image
  </img>
</div>
`;
